packages/components/src/card/body.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { BodyUI } from './styles/card-styles';
+import { useCardContext } from './context';
+
+export const defaultProps = {
+	isShady: false,
+	size: 'medium',
+};
+
+export function CardBody( props ) {
+	const { className, isShady, ...additionalProps } = props;
+	const mergedProps = { ...defaultProps, ...useCardContext(), ...props };
+	const { size } = mergedProps;
+
+	const classes = classnames(
+		'components-card-body',
```

Comment:
One more note, we have these coding guidelines for how class names should be shaped: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#css

I don't think it takes into account the folder structure you opted for. If we would follow the shared guidelines, the name of the class would be:
`components-card__body`

This is why we usually put exposed components in the top folder of the package to have the class name as you coded:
`packages/components/src/card-body/index.js`

I don't feel strongly about it in this case, but I'm sharing for the full picture. I bet we don't follow it for all components as well. However, it would be great to have a good strategy moving forward for new components for consistency.

Reply:
Cool, thanks it looks great now 👍